adder.js

```diff
@@ -1 +1,15 @@
-// code here
+function add(num1, num2) {
+  // num1 = parseInt(prompt("something"));
+  // num2 = parseInt(prompt("something"));
+  return num1 + num2;
+}
+function print(num) {
```

Comment:
the adder function was to add two number 
and it work really fine with strings as well.
Do you remember ? you helped me to do this syntax 
like passing a function as an parameter
I get what you mean. to be honest I really find it struggling to construct my syntax nicely